#!/usr/bin/python
PACKAGE = 'recognition'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('map_size_m',           double_t, 0, 'the size of the map in [m]', 30.0, 1.0, 100.0)
gen.add('map_size_pix',         int_t,    0, 'the size of the map in [pix]', 752, 128, 2048)
gen.add('image_width',          int_t,    0, 'the size of kinect images', 240, 32, 512)
gen.add('image_refresh_span',   double_t, 0, 'the kinect image refresh span [msec]', 0.5, 0.0, 5.0)

exit(gen.generate(PACKAGE, 'recognition', 'RecognitionVisualization'))
